### PR TITLE
chores: clean the code to make it more concise

### PR DESCRIPTION
**Pull request by dima-safin-2025**

### CODE REVIEW

File: src/utils/string.ts [hunk 1]
```diff
@@ -4,16 +4,15 @@ import { strip0x } from '@hyperlane-xyz/utils';
 // Only allows letters and numbers
 const alphanumericRgex = /[^a-zA-Z0-9]/gi;
 export function sanitizeString(str: string) {
-  if (!str || typeof str !== 'string') return '';
```
Comment (jmrossy) [inline src/utils/string.ts:7]: Can please you restore the `typeof` check here?

**Review by jmrossy — COMMENTED** [review]
Nice cleanup, thank you! Just one thing to fix before merging

File: src/utils/string.ts (hunk 1 above)
Comment (dima-safin-2025) [reply src/utils/string.ts:7]: Of course, but I have a question . If  the parameter type of "str" is string in typescript , does typeof str !== 'string' always evaluate to false? 
32c1789e-9245-443f-ad4a-70d0db2d6368

Comment (jmrossy) [reply src/utils/string.ts:7]: Not necessarily. Typescript only does **build-time** checking. At **runtime**, it's still possible due to developer mistakes or invalid data somewhere for a non-string value to get passed into this function. That's why this defensive check is still useful :)

Comment (dima-safin-2025) [reply src/utils/string.ts]: I got it. I have already fixed this issue. Thanks for your patience in answering my questions.

**Review by jmrossy — APPROVED** [review]

**Comment by jmrossy** [comment]
@dima-safin-2025 I've approved the changes but it's look like you'll need to run `yarn prettier --fix`. The code formatter has not been run on your changes.

**Comment by dima-safin-2025** [comment]
> @dima-safin-2025 I've approved the changes but it's look like you'll need to run `yarn prettier --fix`. The code formatter has not been run on your changes.

I've run `yarn prettier --fix` on the affected files and committed the changes

**Pull request merged**